### DISCUSSION TITLE
AI medics do not consume vanilla FAK

### DISCUSSION THREAD

**Discussion by acknowledge14** [acemod/ACE3#8975] [Q&A] [answered]
Hi, I notice in ACE3 Medical System, AI docs do not consume vanilla first aid kits when they heal wounded personnels.
Is that intentional?

**Comment by Drofseh** [comment, marked as answer]
Yes, this is intentional

**Comment by Holgerhollebolle** [comment]
I would prefer if AI Medics consume Items. It would make Medic Logistics a thing.
